docs/collateral-contract.md

```
@@ -0,0 +1,265 @@
+# Collateral Contract
+
+The contract is original from the https://github.com/Datura-ai/celium-collateral-contracts/. Basilica also use the contract to ask miner to make sure their service by deposit collateral to the contract.
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)